packages/financial-templates-lib/src/price-feed/CreatePriceFeed.js

```diff
@@ -376,6 +379,22 @@ async function createPriceFeed(logger, web3, networker, getTime, config) {
       config.priceFeedDecimals, // Defaults to 18 unless supplied. Informs how the feed should be scaled to match a DVM response.
       config.lookback
     );
+  } else if (config.type === "ethvix") {
+    logger.debug({
+      at: "createPriceFeed",
+      message: "Creating EthVixPriceFeed",
+      config
+    });
+
+    return new ETHVIXPriceFeed(
+      logger,
+      web3,
+      config.inverse,
```

Comment:
Is `inverse` a required param?

Reply:
No, because if not provided it would be the long variant (ethVIX) by default. However I could see an argument for making it a required boolean field. What's your opinion?